sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -85,20 +97,48 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
                                           code=lambda_.Code.from_asset(os.path.join(os.path.dirname(os.path.realpath(__file__)), "SDSCode")),
                                           handler="indexer.lambda_handler",
                                           role=lambda_role,
-                                          runtime=lambda_.Runtime.PYTHON_3_9,
+                                          runtime=lambda_.Runtime.PYTHON_3_7,
```

Comment:
Is there reason why we went down to python3.7 instead of python3.9?

Reply:
for some reason when I use the lambda_python_alpha library for layers, it doesn't allow me to use a python version greater than 3.7. I'm going to add an issue for this to spend more time trying to figure out why.

Reply:
This issue was fixed by a suggestion in another comment. The lambdas are now using 3.9